app/controllers/application_controller.rb

```diff
@@ -91,8 +91,13 @@ def save_and_audit(model, attributes)
     def check_authorization
       if !@incident.authorized_to_view?(@current_user)
         raise BridgeExceptions::UnathorizedToView.new
-      elsif !@incident.authorized_to_edit?(@current_user) && params[:action] != 'review'
-        redirect_to review_incident_path(@incident)
+      elsif !@incident.authorized_to_edit?(@current_user)
+        # Redirect to review page if possible.
```

Comment:
I can write a test for the viewable+uneditable+incomplete case, but it would have to be a pretty pathological situation to trigger this kind of behavior.

One situation that could cause it:
1. an incident gets submitted to state,
2. a DOJ user edits it *after* submission but accidentally makes the incident invalid,
3. an admin user tries to access that same incident from the Past Submission page.

I made this change to prevent a redirect loop from arising in this scenario, but, that said, this is a pretty unlikely situation, and I dunno if it makes sense to write a request spec for it.